src/littlefs/context.py

```diff
@@ -74,3 +75,86 @@ def sync(self, cfg: 'LFSConfig') -> int:
             Filesystem configuration object
         """
         return 0
+
+
+
+# if the user has win32file installed then define this class 
+try:  
+    import win32file  
+except ImportError:
+    raise ImportError("Unable to import 'win32file'. This module is required for Windows-specific functionality. Please ensure you are running on a Windows platform or install 'pywin32' using: 'pip install pywin32'.")
```

Comment:
so hold on, this now makes `win32file` a hard requirement. We should only raise an error if the user attempts to instantiate `UserContextWinDisk`. The following is probably closer to what we want:

```
try:
    import win32file
except ImportError:
    win32file = None

class UserContextWinDisk(UserContext):
     def __init__(self, disk_path:str) -> None:
         if win32file is None:
             raise ImportError(...)
         # rest of WinDisk implementation
```